src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,34 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classname from 'classnames';
+import { TodoWithUser } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+import { User } from '../../types/User';
+
+type Props = {
+  todo: TodoWithUser,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    id,
+    completed,
+    title,
+    user,
+  } = todo;
+
+  return (
+    <article
+      key={id}
+      data-id={id}
+      className={classname('TodoInfo', {
+        'TodoInfo--completed': completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+
+      <UserInfo user={user as User} />
```

Comment:
```suggestion
      {user && <UserInfo user={user} />}
```
Don't do this, if the user is `null`, don't render the UserInfo component